packages/block-library/src/embed/embed-placeholder.native.js

```diff
@@ -8,12 +8,25 @@ import { TextInput } from 'react-native';
  */
 import { __ } from '@wordpress/i18n';
 
+/**
+ * Internal dependencies
+ */
+import EmbedBottomSheet from './embed-bottom-sheet';
+
 const EmbedPlaceholder = ( { value } ) => {
 	return (
-		<TextInput
-			value={ value }
-			placeholder={ __( 'Enter URL to embed here…' ) }
-		/>
+		<>
+			<TextInput
+				value={ value }
+				placeholder={ __( 'Enter URL to embed here…' ) }
+			/>
+			<EmbedBottomSheet
```

Comment:
Since the https://github.com/WordPress/gutenberg/pull/30592 has been merged in, I will give this PR another review when the `EmbedBottomSheet` is migrated there.

Reply:
I've just updated the PR with `trunk` (which includes the changes from #30592) and applied some updates.